samples/bluetooth/iso_broadcast/prj.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_BT=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_ISO=y
+CONFIG_BT_ISO_BROADCAST=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_DEVICE_NAME="Test ISO Broadcaster"
+
+# Temporary, enable the following to meet BT_ISO dependencies
+CONFIG_BT_OBSERVER=y
```

Comment:
Why is this needed?

Reply:
Right. The issue is that iso broadcast support for receiver requires this and we only have a aggregated Kconfig for iso broadcast support. There's a TODO to split it into broadcaster and receiver, so I guess that needs to be done before we can fix these :) I guess that's on me ^_^